pkg/cmd/server/api/types.go

```diff
@@ -296,19 +296,19 @@ type MasterConfig struct {
 
 // JenkinsPipelineConfig holds configuration for the Jenkins pipeline strategy
 type JenkinsPipelineConfig struct {
```

Comment:
for other optional portions of the config, we made the parent struct a pointer... was there a reason not to do that here?